applications/nublado/values-base.yaml

```diff
@@ -0,0 +1,78 @@
+controller:
+  config:
+    images:
+      source:
+        type: "docker"
+        registry: "ts-dockerhub.lsst.org"
+        repository: "sal-sciplat-lab"
+      num_releases: 0
+      num_weeklies: 3
+      num_dailies: 2
+      cycle: 31
+      recommended_tag: "recommended_c0031"
+    lab:
+      env:
+        DAF_BUTLER_REPOSITORY_INDEX: "/project/data-repos.yaml"
+        LSST_DDS_INTERFACE: "net1"
+        LSST_DDS_PARTITION_PREFIX: "base"
+        LSST_SITE: "base"
+        PGPASSFILE: "/opt/lsst/software/jupyterlab/butler-secret/postgres-credentials.txt"
+        PGUSER: "oods"
+      volumes:
+      - containerPath: "/home"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/jhome"
+          server: "nfs-jhome.ls.lsst.org"
+      - containerPath: "/project"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/project"
+          server: "nfs-project.ls.lsst.org"
+      - containerPath: "/scratch"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/scratch"
+          server: "nfs-scratch.ls.lsst.org"
+      - containerPath: "/datasets"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/lsstdata"
+          server: "nfs-lsstdata.ls.lsst.org"
+      - containerPath: "/repo/LATISS"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/auxtel/repo/LATISS"
+          server: "nfs-auxtel.ls.lsst.org"
+      - containerPath: "/net/obs-env"
+        mode: "rw"
+        source:
+          type: "nfs"
+          path: "/obs-env"
+          server: "nfs-obsenv.ls.lsst.org"
+      - containerPath: "/data/lsstdata/BTS/auxtel"
+        source:
+          type: "nfs"
+          path: "/auxtel/lsstdata/BTS/auxtel"
+          server: "nfs-auxtel.ls.lsst.org"
+
+jupyterhub:
+  cull:
+    users: false
+    removeNamedServers: false
+    timeout: 432000
+    every: 300
+    maxAge: 2160000
+  singleuser:
+    extraAnnotations:
+      k8s.v1.cni.cncf.io/networks: "kube-system/dds"
```

Comment:
Is this supposed to be an additional annotation to add to the created pod? If so, does this work? I feel like it couldn't work, since the Nublado controller is creating the pod and it doesn't know anything about this setting.

Reply:
Uh oh.  Something like this is going to need to work, which may require adding a new config setting to the controller.

Yes, this is the magic annotation on the pod that causes it to couple to the correct multus-provided network.  It is untested, because only T&S sites use this feature.

Reply:
OK, it we pass "annotations" into create_pod() it will do the right thing.  Not sure if we've already got annotations in the pod config, but if not it's easy to add.

Reply:
Yeah, we'll have to add it as a config setting and then pipe it down through create_user_pod

Reply:
cf https://github.com/lsst-sqre/jupyterlab-controller/pull/148

Reply:
I've done that now.